pyfar/dsp/dsp.py

```diff
@@ -1985,6 +1995,14 @@ def average(signal, mode='linear', caxis=None, weights=None, keepdims=False,
             f"mode is '{mode}' and signal is type '{signal.__class__}'"
             " but must be of type 'Signal' or 'FrequencyData'."))
 
+    if ((type(signal) is pyfar.TimeData or type(signal) is pyfar.Signal)
```

Comment:
I understand this for TimeData, but why are the modes not defined for complex Signals?

Reply:
Sorry for that. At the very beginning we said it. But with the last discussion this mode restriction is not necessary anymore. I removed it